src/main/java/edu/hawaii/its/api/service/GroupingsService.java

```diff
@@ -25,15 +31,50 @@
 @Service
 public class GroupingsService {
 
+    @Value("${groupings.api.grouping_admins}")
+    private String GROUPING_ADMINS;
+
     @Value("${groupings.api.trio}")
     private String TRIO;
 
+    public static final Log logger = LogFactory.getLog(GroupingAssignmentService.class);
```

Comment:
The class for this should be `GroupingsService.class`

- [ ] Replace `GroupingAssignmentService.class` with `GroupingService.class`